src/main/java/com/conveyal/datatools/manager/DataSanitizer.java

```diff
@@ -159,12 +255,7 @@ private static int deleteOrphanedFeedVersions(List<FeedVersion> feedVersions) {
     public static int deleteOrphanedDBSchemas(Set<String> orphanedSchemas) {
         int deletedSchemas = 0;
         for (String orphanedSchema : orphanedSchemas) {
-            try {
-                GTFS.delete(orphanedSchema, DataManager.GTFS_DATA_SOURCE);
-                LOG.info("Dropped orphaned DB schema from Postgres.");
-            } catch (SQLException | InvalidNamespaceException e) {
-                System.err.printf("Failed to delete DB schema: %s. %s%n", orphanedSchema, e.getMessage());
-            }
+            FeedVersion.deleteDBSchema(orphanedSchema);
```

Comment:
L286 below: could you remove the `LOG.info` statement, please? It prints a massive list that is not relevant to the orphaned schema removal.